### DISCUSSION TITLE
Primefaces grouped bar chart values and percentages

### DISCUSSION THREAD

**Discussion by plyonlee** [primefaces/community#1080] [PrimeFaces] [answered]
I have a grouped bar chart that shows groups of information, the groups are made up of 2 elements the first is the number of departures and the second is the number of departures that where on time. 

My issue is that for the tool tip I want to add in a third piece of information showing a percentage it is for example
total departures: 200
on time departures:150
percentage: 75%

is there a way to do this? I have searched high and low and I'm not getting anywhere.

**Comment by melloware** [comment, marked as answer]
Yep: https://www.chartjs.org/docs/latest/samples/tooltip/content.html

Or: https://www.chartjs.org/docs/latest/samples/tooltip/html.html

And just use the ChartJS extender to set that tooltip:

https://primefaces.github.io/primefaces/13_0_0/#/components/charts?id=extender
